pkg/app/api/service/pipedservice/service.proto

```diff
@@ -525,5 +527,8 @@ message InChainDeploymentPlannableRequest {
 }
 
 message InChainDeploymentPlannableResponse {
+    // plannable used to determine whether piped should start planning the given development.
     bool plannable = 1;
+    // cancel used to determine whether piped should cancel the given development.
+    bool cancel = 2;
```

Comment:
Maybe we should have `cancel_reason` to show on the UI instead of passing `""` while calling `cancelDeployment` as the current implementation.

Reply:
Nice point, addressed with latest commit 👍